### PR TITLE
Closes Agency Person homepage (Case Manager #230)

### PR DESCRIPTION
**Pull request by sidhene**

### CODE REVIEW

**Comment by sidhene** [comment]
The failing test is the one I got help on in PL today, but apparently I did not understand the answer.

File: spec/controllers/people_invitations_controller_spec.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@
       end
       it 'redirects as specified' do
         expect(response).
-            to render_template(@controller.after_invite_path_for(agency_admin))
+            to render_template(devise/mailer/invitation_instructions)
```
Comment (patmbolger) [inline spec/controllers/people_invitations_controller_spec.rb]: the argument to render_template should be a string (see line 205 in same file)

**Comment by patmbolger** [comment]
You should rebase against upstream development and then push to your repo again .... your PR includes 140 files!

**Comment by sidhene** [comment]
@patmbolger - I'm going to need to pair on the rebase: http://stackoverflow.com/questions/36638043/i-cant-believe-git-rebase-is-supposed-to-work-this-way

File: spec/controllers/people_invitations_controller_spec.rb (hunk 1 above)
Comment (sidhene) [inline spec/controllers/people_invitations_controller_spec.rb]: Ok Thanks.

**Pull request closed without merging**